### ISSUE TITLE
Favourites Fragment 

### ISSUE THREAD

**Issue by tanmaydixit** [MobileApplicationsClub/DoJMA-iOS#4]
The item on the herald list will have a favorite(heart) button, this will become red if someone clicks it.
Then the list item(herald) article will come on the favorites fragment. Only the ones marked heart by the user will be shown out here(the List on the Favourites fragment).
The data that the user has chosen this as fav or not will be stored locally.

**Comment by aryanagarwal15** [comment]
@tanmaydixit I don't think we need a favourite's tab made for DoJMA App. I have not seen any person using this on DoJMA's android app. Lets just keep 4 tabs in the bottom nav bar in iOS.
